service/awsconfig/v4/resource/cloudformation/spec.go

```diff
@@ -23,10 +24,12 @@ const (
 
 // StackState is the state representation on which the resource methods work.
 type StackState struct {
-	Name                     string
+	Name string
+
 	MasterImageID            string
 	MasterInstanceType       string
 	MasterCloudConfigVersion string
+
```

Comment:
Took me since Friday to realize that there is a pattern between master and worker state. Made blocks for visual support.